README.md

```diff
@@ -155,21 +155,19 @@ it will automatically re-render when the form changes.
 
 ### B) Use [Symfony UX Turbo](https://symfony.com/bundles/ux-turbo/current/index.html#decomposing-complex-pages-with-turbo-frames)
 
-By using turbo frames you can have a dynamic form running quickly without any JavaScript.
+If you are already using Symfony UX Turbo on your website, you can have a dynamic form running quickly without any JavaScript.
+
+Or you may want to install Symfony UX Turbo, [check out the documentation](https://symfony.com/bundles/ux-turbo/current/index.html#installation).
+
+> [!NOTE]
+> You only need to have Turbo Frame, you can disable Turbo Drive if you do not use it, or do not want to use it.
+> ie: `Turbo.session.drive = false;`
 
 Simply add a `<turbo-frame>` around your form:
 
 ```twig
 <turbo-frame id="rating-form">
-{{ form_start(form) }}
-    {{ form_row(form.rating) }}
-
-    {% if form.badRatingNotes is defined %}
-        {{ form_row(form.badRatingNotes) }}
-    {% endif %}
-    {# We also add a real submit button #}
-    {{ form_row(form.submit) }}
-{{ form_end(form) }}
+{{ form(form) }}
```

Comment:
I like this form simplification example. The only point we want to mention here - users need to render the form. How - that's up to them, and Symfony docs about Forms help with it.

But let's indent it.
```suggestion
    {{ form(form) }}
```